testing/scripts/README.md

```diff
@@ -1,11 +1,15 @@
 # End to End Tests
 
-## Prerequisites
+## Running the test suite
```

Comment:
I don't think this page is currently part of the docs, maybe we should also link it on the docs. Also the CONTRIBUTING.md has a lot of useful content but it's not linked. We could add it under the Developer section in the docs. @adriangonz whats your thoughts?

Reply:
That's a very good suggestion! I will add a small section to `CONTRIBUTING.md` on "integration tests" (pointing to `testing/scripts/README.md`) and then add `CONTRIBUTING.md` to the main SC docs.